Packs/Microsoft365Defender/Playbooks/Microsoft_365_Defender_-_Emails_Indicators_Hunt.yml

```diff
@@ -0,0 +1,858 @@
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      IPAddress Exist:
+      - "14"
+    separatecontext: false
+    conditions:
+    - label: IPAddress Exist
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.IPAddress
+            iscontext: true
+          right:
+            value: {}
+    view: |-
+      {
+        "position": {
+          "x": 1450,
+          "y": 270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: 5270e0e4-c680-4bb8-85a1-3a47b07cf5cd
+    type: regular
+    task:
+      id: 5270e0e4-c680-4bb8-85a1-3a47b07cf5cd
+      version: -1
+      name: Set IPAddress search argument
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+      - "18"
+    scriptarguments:
+      key:
+        simple: IPAddressParsed
+      value:
+        complex:
+          root: inputs.IPAddress
+          transformers:
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ', '
+          - operator: replace
+            args:
+              limit: {}
+              replaceWith:
+                value:
+                  simple: '", "'
+              toReplace:
+                value:
+                  simple: ', '
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1450,
+          "y": 510
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: 66fb9f8e-164e-48f4-8e87-c2be4322639e
+    type: regular
+    task:
+      id: 66fb9f8e-164e-48f4-8e87-c2be4322639e
+      version: -1
+      name: Retrieve emails data (IPAddress)
+      description: 'Advanced hunting is a threat-hunting tool that uses specially
+        constructed queries to examine the past 30 days of event data in Microsoft
+        365 Defender. Details on how to write queries: https://docs.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-query-language?view=o365-worldwide.'
+      script: Microsoft 365 Defender|||microsoft-365-defender-advanced-hunting
+      type: regular
+      iscommand: true
+      brand: Microsoft 365 Defender
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      limit:
+        complex:
+          root: inputs.ResultsLimit
+      query:
+        simple: let _start = now(-${inputs.SearchTimeframe}d); EmailEvents | where
```

Comment:
Whats the difference between task 15 and task 18?

Reply:
Two different tables with different data. One is to search whether the ip is a part of a url and the other is to check the sender ip.

Reply:
Actually I can see now that I need to change the "project away" fields for one of them